GVRf/Framework/src/org/gearvrf/GVRScene.java

```diff
@@ -152,6 +153,57 @@ public void setFrustumCulling(boolean flag) {
     public void setOcclusionQuery(boolean flag) {
         NativeScene.setOcclusionQuery(getNative(), flag);
     }
+
+    private GVRConsole mStatsConsole = null;
+    private boolean mStatsEnabled = false;
+
+    /**
+     * Returns whether displaying of stats is enabled for this scene.
+     * 
+     * @return whether displaying of stats is enabled for this scene.
+     */
+    public boolean getStatsEnabled() {
+        return mStatsEnabled;
+    }
+
+    /**
+     * Set whether to enable display of stats for this scene.
+     *
+     * @param enabled
+     *     Flag to indicate whether to enable display of stats.
+     */
+    public void setStatsEnabled(boolean enabled) {
+        mStatsEnabled = enabled;
+
+        if(enabled && mStatsConsole == null) {
+            mStatsConsole = new GVRConsole(getGVRContext(), GVRConsole.EyeMode.BOTH_EYES);
+            mStatsConsole.setXOffset(300.0f);
+            mStatsConsole.setYOffset(300.0f);
+        }
+
+        if(enabled && mStatsConsole != null) {
+            mStatsConsole.setEyeMode(GVRConsole.EyeMode.BOTH_EYES);
+        } else if(!enabled && mStatsConsole != null) {
+            mStatsConsole.setEyeMode(GVRConsole.EyeMode.NEITHER_EYE);
+        }
+    }
+
+    void resetStats() {
+        if(mStatsEnabled) {
+            mStatsConsole.clear();
+            NativeScene.resetStats(getPtr());
+        }
```

Comment:
One more small concern is like you should use getNative() instead of getPtr() since getPtr() is an old version API.